lib/probes/generic-pool.js

```diff
@@ -5,14 +5,12 @@ const ao = require('..')
 const log = ao.loggers
 
 const semver = require('semver')
-const nodeVersion = semver.major(process.version)
-
 const logMissing = ao.makeLogMissing('generic-pool')
 
 module.exports = function (gp, info) {
   const version = info.version
   const majorVersion = semver.major(version)
-  if (majorVersion >= 3 && nodeVersion >= 8) {
+  if (majorVersion >= 3) {
     // version 3 is a major overhaul. only patch it if running node version 8.
```

Comment:
remove `only patch it if running node version 8.`

Reply:
Not supporting anything under 14, thus not checking for unsupported configurations, accepting that with unsupported configurations, unexpected things are expected.

Reply:
I meant to update the comment, it is a bit confusing to mention version 8 when there is no action regarding version 8 anymore ;)

Reply:
I've created a pull request to fix that! https://github.com/appoptics/appoptics-apm-node/pull/205